Src/library/base/elks/kernel/file.e

```diff
@@ -313,6 +313,44 @@ feature -- Access
 			Result := c_retrieved (descriptor)
 		end
 
+	null_path: PATH
+			-- Null device path.
+		note
+			EIS: "name=Null Device", "src=https://en.wikipedia.org/wiki/Null_device", "protocol=uri"
+		do
+			-- TODO check vxWorks null device.
```

Comment:
you should use the `null_name`....

`null_path: PATH do create Result.make_from_string (null_name)`

Question: I guess STRING for null_name is ok, but would it hurt to have STRING_32 ?

Reply:
Ok done
About STRING or STRING_32 for `null_name`, initially I used STRING_32 but then I change to STRING since the current feature `name` is using STRING.